Client.Linq/Internal/QueryAggregator.cs

```diff
@@ -74,6 +75,7 @@ internal QueryAggregator()
             _limitTailNOffsetAssignments = new List<LimitOffsetAssignment>();
             _filterByTags = new List<string>();
             _filterByFields = new List<string>();
+            _imports = null;
```

Comment:
Initialise to:

```suggestion
            _imports = new HashSet<string>();
```

Reply:
Why? so we have a worse memory footprint